### PR TITLE
chore(Components): Add Flow typing to LatexRenderer & all transforms components

### PR DESCRIPTION
**Pull request by peggyrayzis**
Added Flow typing to all components that live in the transforms folder & the LatexRenderer component

### CODE REVIEW

File: src/notebook/components/latex.js [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
 import React from 'react';
-import PureRenderMixin from 'react-addons-pure-render-mixin';
+import { shouldComponentUpdate } from 'react-addons-pure-render-mixin';
 import { loadMathJax, typesetMath } from 'mathjax-electron';
 
+type Props = {
+  children?: React.Element<any>,
+};
+
+const MathJax: global = window.MathJax;
```
Comment (rgbkrk) [inline src/notebook/components/latex.js:10]: Uh oh, now you're finding our wartiest code. :wink:

**Comment by rgbkrk** [comment]
Excellent work here.

Would you mind [rebasing interactively](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) to get rid of the merge commit and merge conflict?

Also, let us know if master builds well for you or not. We recently switched to a process that demands more upfront tooling installed. :worried:

File: src/notebook/components/latex.js (hunk 1 above)
Comment (peggyrayzis) [reply src/notebook/components/latex.js:10]: Hahaha it was only a matter of time until I found some 😆

**Comment by peggyrayzis** [comment]
Ugh I tried rebasing but the build is still saying I have a conflict even though I fixed it...help? 🙈

**Comment by peggyrayzis** [comment]
The master build was failing yesterday until I read the updated readme. Brew installing all the new dependencies & then npm install fixed any problems for me :)

**Comment by rgbkrk** [comment]
The easiest option is to have me do the rebase and then post about what steps I took.

I can either do the rebase, force push the repo, and have you pull _or_ I can just post my steps to follow/learn from.

**Comment by rgbkrk** [comment]
Rebasing is one of the most persnickety git flows, I hate asking it of people but it's also super useful to know. Thanks for rolling through it.

**Comment by rgbkrk** [comment]
Wow this rebase is hairy in the `json.js` component.

**Comment by peggyrayzis** [comment]
Yeah, that's why I needed a second pair of eyes :/ I know how to rebase, git just refuses to accept that the conflicts are fixed after I resolve them and rebase --continue. I'd love to see how you fix it!

**Comment by rgbkrk** [comment]
screen shot 2016-10-10 at 1 04 34 pm
screen shot 2016-10-10 at 12 27 36 pm
screen shot 2016-10-10 at 12 27 50 pm
screen shot 2016-10-10 at 12 27 55 pm

git wants to me to see that I need to resolve and then `git add`

screen shot 2016-10-10 at 12 28 24 pm
screen shot 2016-10-10 at 12 29 57 pm
screen shot 2016-10-10 at 12 30 13 pm

Missing part here:

```
git add src/notebook/components/notebook.js
git rebase --continue
```

screen shot 2016-10-10 at 12 31 37 pm
screen shot 2016-10-10 at 12 31 45 pm
screen shot 2016-10-10 at 12 32 15 pm
screen shot 2016-10-10 at 12 36 55 pm
screen shot 2016-10-10 at 12 37 05 pm
screen shot 2016-10-10 at 12 37 11 pm

**Comment by rgbkrk** [comment]
I might be missing some screenshots in there. ¯\_(ツ)_/¯

**Comment by rgbkrk** [comment]
Oh I also didn't show that I did a `git fetch origin` (noting that my origin is this repo).

**Comment by peggyrayzis** [comment]
Awesome, thank you!! I'll look over these steps. Did you force push the rebase? I don't see the build restarting.

**Comment by rgbkrk** [comment]
> Yeah, that's why I needed a second pair of eyes :/ I know how to rebase, git just refuses to accept that the conflicts are fixed after I resolve them and rebase --continue. I'd love to see how you fix it!

Sorry! I didn't mean to overexplain or step on your toes. I wanted to make sure that if there was anything I could help teach for the future, I wanted to be able to.

**Comment by rgbkrk** [comment]
Went ahead and force pushed.

**Comment by peggyrayzis** [comment]
> Sorry! I didn't mean to overexplain or step on your toes. I wanted to make sure that if there was anything I could help teach for the future, I wanted to be able to.

No worries, I appreciate all the help!! 👍 

I'll fix the linting errors.

**Comment by rgbkrk** [comment]
Great! Thank you so much!

Note for the future: we don't merge our own PRs in nteract

**Comment by peggyrayzis** [comment]
Sorry, I'll make a note of that for next time :)

**Pull request merged**